packages/lib/server/service/insightsBooking.ts

```diff
@@ -92,8 +129,6 @@ export class InsightsBookingService {
         COUNT(*)::int as "count"
       FROM "BookingTimeStatusDenormalized"
       WHERE ${baseConditions}
-        AND "startTime" >= ${startDate}::timestamp
-        AND "startTime" <= ${endDate}::timestamp
         AND "status" = 'accepted'
```

Comment:
this `startTime` check has been moved into `filters`. The following is added on the trpc-router.ts file.

```
          dateRange: {
            target: "startTime",
            startDate,
            endDate,
          },
```